packages/neuron-ui/src/services/remote/index.ts

```diff
@@ -1,12 +1,12 @@
+import { ipcRenderer, desktopCapturer, shell, OpenDialogOptions, MenuItemConstructorOptions, MenuItem } from 'electron'
```

Comment:
I see they are still injected from the preload script, will we use both of  them?

Reply:
Yes. see https://github.com/nervosnetwork/neuron/pull/1802 PR description for details.

Reply:
Yes, see https://github.com/nervosnetwork/neuron/pull/1802 PR description for details.